Sprint-1/interpret/percentage-change.js

```diff
@@ -12,11 +12,26 @@ console.log(`The percentage change is ${percentageChange}`);
 // Read the code and then answer the questions below
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
+// Number(carPrice.replaceAll(",", "")) — This line calls two functions: .replaceAll() and Number().
+// Number(priceAfterOneYear.replaceAll(",", "")) — This line also calls two functions: .replaceAll() and Number().
```

Comment:
What about ``console`` in line 10`?

Reply:
Hi Maesierra,

Thank you for your review! Do you mean I didn't write any comments for console.log? If so, I will add it. 😊